pkg/volume/rbd/rbd_util.go

```diff
@@ -789,3 +795,15 @@ func (util *RBDUtil) rbdStatus(b *rbdMounter) (bool, string, error) {
 		return false, output, nil
 	}
 }
+
+// getRbdImageInfo try to get RbdImageInfo from deviceMountPath.
+func getRbdImageInfo(deviceMountPath string) (*RbdImageInfo, error) {
```

Comment:
use lower case (do not export types), or return two plain variables.

Reply:
ok, done as you suggested